internal/service/uow/transaction_test.go

```
@@ -15,422 +18,1752 @@
+	tests := []struct {
+		name           string
+		createSvc      func(t *testing.T, systemDriver *mocks.MockDriver) *Service
+		createRequests func(storageDriver storage.Driver) map[storage.Driver]*storage.Request
+		createTx       func(storageDriver storage.Driver) (*storage.Transaction, error)
+		checkTx        func(t *testing.T, tx *storage.Transaction, driver storage.Driver)
+		setupMocks     func(t *testing.T, driver *mocks.MockDriver, systemDriver *mocks.MockDriver)
+		wantErr        require.ErrorAssertionFunc
+	}{
+		{
+			name: "positive case",
+			createSvc: func(t *testing.T, systemDriver *mocks.MockDriver) *Service {
```

golangci-lint (2.5.0)

[error] test helper function should start from t.Helper()

(thelper)